backend/spec/graphql/types/query_type_spec.rb

```diff
@@ -68,4 +68,41 @@
       expect(result.dig('data', 'journal', 'id')).to eq(journal_id.to_s)
     end
   end
+
+  describe 'users' do
+    before do
+      create_list(:user, 3)
+    end
+
+    let(:query) do
+      <<~QUERY
+        query GetAllUsers {
+          users {
+            id
+            name
+            nickname
+            email
+          }
+        }
+      QUERY
+    end
+
+    let(:result) do
+      BackendSchema.execute(query)
+    end
+
+    it '全てのuserを取得できる' do
+      expect(result.dig('data', 'users').size).to eq(User.count)
+    end
+
+    it '任意のuserについてモデルと一致する内容を取得できる' do
+      random_user = User.all.sample
+      expect(result.dig('data', 'users').find { |user| user['id'] == random_user.id.to_s }).to eq(
+        'id' => random_user.id.to_s,
+        'name' => random_user.name,
+        'nickname' => random_user.nickname,
+        'email' => random_user.email
+      )
```

Comment:
ちょっと観点外ですが、expectの中でfindしてるので「結局何が検査対象なの？」っていうのが若干読み取りづらかったので、好みもありますが一度ローカル変数に落としても良いと思います。

```suggestion
      actual = result.dig('data', 'users').find { |user| user['id'] == random_user.id.to_s }
      expect(actual['id']).to eq(random_user.id_to_s)
      ...
```

こんな感じですかね？
(ruby全然書いてないので適当でごめんなさい伝われこのイメージ)

Reply:
同意です！
@DaisukeShinoku ご確認いただいて、必要な分取り込みと修正お願いします👍
シンオクさんのおかげで学びが深まりました🙏